synaptic_reconstruction/tools/util.py

```diff
@@ -50,7 +65,14 @@ def get_model(model_type: str, device: Optional[Union[str, torch.device]] = None
     """
     if device is None:
         device = get_device(device)
-    model_path = get_model_path(model_type)
+
+    model_registry = get_model_registry()
+    model_path = model_registry.fetch(model_type)
+    warnings.filterwarnings(
```

Comment:
Please remove the filter. It's only relevant for the current torch version and should not be suppressed.